pom.xml

```diff
@@ -17,19 +17,17 @@
     <modules>
         <module>quarkus-opentelemetry-exporter-jaeger</module>
         <module>docs</module>
+        <module>quarkus-opentelemetry-exporter-gcp</module>
```

Comment:
We also need documentation here: https://github.com/quarkiverse/quarkus-opentelemetry-exporter/tree/main/docs/modules/ROOT/pages

Reply:
Done.